src/components/theme-switch/user-dropdown-switcher/index.tsx

```diff
@@ -0,0 +1,20 @@
+import ThemeSwitcher from 'components/theme-switch'
+import {
+	DropdownDisclosureListItem,
+	DropdownDisclosureSeparatorItem,
+} from 'components/dropdown-disclosure'
+import s from './user-dropdown-switcher.module.css'
+
+export default function UserDropdownDisclosureThemeSwitcher() {
+	return (
+		<>
+			<DropdownDisclosureSeparatorItem />
+			<DropdownDisclosureListItem className={s.listItem}>
+				<label className={s.label} htmlFor="theme-switcher">
+					Theme
+				</label>
```

Comment:
I'm unsure if I should still use this as a `label` since I added an `aria-label` as well. Open to feedback here.